debian/clearwater-etcd.init.d

```diff
@@ -199,8 +208,9 @@ join_cluster()
 #
 join_or_create_cluster()
 {
-        if [[ $etcd_cluster == $advertisement_ip ]] ||
-           [[ $etcd_cluster =~ (^|,)$advertisement_ip(,|$) && ! -f $JOINED_CLUSTER_SUCCESSFULLY ]]
+        # We only want to create the cluster if we are both a founding member,
+        # and we have never successfully clustered before. Otherwise, we join
+        if [[ ! -f $JOINED_CLUSTER_SUCCESSFULLY && $etcd_cluster =~ (^|,)$advertisement_ip(,|$) ]]
```

Comment:
It does not (just tested). 
What exactly do we expect as the value of etcd_cluster?
i.e., should i simply add a match on a space preceding $advertisement_ip, or do we need to allow a space on either side?